graphs/omen-thumbnailmapping/src/schema.ts

```diff
@@ -0,0 +1,81 @@
+import {
+  Entity,
+  Value,
+  ValueKind,
+  store,
+  Bytes,
+} from "@graphprotocol/graph-ts";
+
+export class OmenThumbnailMapping extends Entity {
+  constructor(marketAddress: Bytes) {
+    super();
+    this.set("marketAddress", Value.fromBytes(marketAddress));
+  }
+
+  save(): void {
+    let marketAddress = this.get("marketAddress");
+    let image_hash = this.get("image_hash");
+    assert(marketAddress != null, "Cannot save OmenThumbnailMapping entity without an marketAddress");
+    assert(image_hash != null, "Cannot save OmenThumbnailMapping entity without an image_hash");
+    if (marketAddress && image_hash) {
+      assert(
+        marketAddress.kind == ValueKind.BYTES,
+        `Entities of type OmenThumbnailMapping must have an marketAddress of type Bytes but the marketAddress '${marketAddress.displayData()}' is of type ${marketAddress.displayKind()}`,
+      );
+      assert(
+        image_hash.kind == ValueKind.BYTES,
+        `Entities of type OmenThumbnailMapping must have an image_hash of type Bytes but the image_hash '${image_hash.displayData()}' is of type ${image_hash.displayKind()}`,
+      );
+      store.set("OmenThumbnailMapping", marketAddress.toBytes().toHexString(), this);
+    }
+  }
+
+  static loadInBlock(marketAddress: Bytes): OmenThumbnailMapping | null {
+    return changetype<OmenThumbnailMapping | null>(
+      store.get_in_block("OmenThumbnailMapping", marketAddress.toHexString()),
+    );
+  }
```

Comment:
Could you please provide more details about this? Not clear to me.

Reply:
Same here! It was auto-generated by graph's CLI 😄